README.md

```diff
@@ -1,45 +1,73 @@
 
-## Creating a project
+Todo:
+- [ ] Write about the database architecture
 
-If you're seeing this, you've probably already done this step. Congrats!
+## API
 
-```bash
-# create a new project in the current directory
-npm create svelte@latest
+The API is split into four different base routes, admin, authed, scout, and submit. All routes shown below are within the /api/ route. All endpoint methods are POST.
 
-# create a new project in my-app
-npm create svelte@latest my-app
-```
+### How to use the API
 
-## Developing
+#### Match Scouting
+1. The admin signs in, their signed-in status is verified by the admin/authed endpoint. 
+2. The admin makes a request to the admin/make-event endpoint to create an empty event.
+3. All the teams need to be loaded into the database, we wrote a python script to grab them all from TBA (write-up coming later).
+4. The following steps can happen asynchronously and must happen for every match in an event.
+5. The admin enters a MatchKey and then enters the teams for that match either by doing it manually or using autofill by querying the admin/teams endpoint. The admin then releases a match with the admin/assign endpoint.
+6. The scouts authenticate with the passphrase, they is verified with the authed endpoint.
+7. The scouts then login and repeatedly query the scout endpoint with a (4 second interval in between queries) until they have been assigned a team to scout.
+8. Once the match is scouted, the scouts submit their results with the submit/match endpoint.
 
-First, make sure to create a secrets folder containing passwords.ts in the root directory. Passwords should look like this:
+### [/scout](/src/routes/api/scout/+server.ts)
+This is very simple, it just exists to poll the next robot from the scouting queue. It takes an AbortController signal as a request param.
 
-```
-export const user_password = "password";
-export const admin_password = "adminpassword";
-```
+This request is made by a client every second or so while they're logged in. As soon as a match is released by the admin, the client will receive a valid response.
 
-Then, once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+### [/authed](/src/routes/api/authed/+server.ts)
+This endpoint is also very simple. It just checks if the passphrase is correct.
 
-```bash
-npm run dev
+### Submit
+This route contains two endpoints, match, pit, and photo. Each one of for a different type of submission. These endpoints are requested when the client wants to submit scouting data.
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
-```
+#### [/submit/match](/src/routes/api/submit/match/+server.ts) 
+This endpoints basically just takes in all the data from a match, including the cycle and defense times, and puts them in the database. [Request made from here](/src/lib/components/match-scout-components/Submit.svelte)
```

Comment:
"basically just" is informal language. This could be simplified to "This endpoint takes in all the data from a ..."

Reply:
```suggestion
This endpoint takes in all the data from a match, including the cycle and defense times, and puts them in the database. [Request made from here](/src/lib/components/match-scout-components/Submit.svelte)
```